flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/SqlBase.java

```diff
@@ -86,7 +86,7 @@ protected void dropDatabase(String database, boolean ifExists) {
     sql("DROP DATABASE %s %s", ifExists ? "IF EXISTS" : "", database);
   }
 
-  protected static String toWithClause(Map<String, String> props) {
+  public static String toWithClause(Map<String, String> props) {
```

Comment:
Why is this change? Did this become a utility method which is shared between tests? Can we find a better place for it then?

Reply:
This is currently being across Tests under org.apache.iceberg.flink. Wanted to use for test under org.apache.iceberg.flink.source. org.apache.iceberg.flink.source.SqlHelpers can be another option, where this can be done.

Reply:
Sounds good